### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by ledogar**
### Description of your changes
Documentation on how to run the tests

### Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation add / update

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,12 @@ Scripts to create in silico lipid fragmentation spectra, based on analysis of ch
 
 3. `python setup.py sdist bdist_wheel`
 
+# Run Tests
+
+1. `pip install pytest`
+
+2. `pytest test.py`
+
```
Comment (PMSeitzer) [inline README.md:17]: Should there be a `test.py` file added to the repo?
Otherwise lgtm

Comment (ledogar) [reply README.md:17]: Yes just cleaned that up

**Comment by ledogar** [comment]
I just added a suggestion to use pypy.  The tests take 6m to complete with pypy, and 28m to complete without it.  What do you think -- too weird to suggest this?

On Mac it's a simple `brew install pypy` then `pypy -m pytest tests`

**Comment by ledogar** [comment]
@PMSeitzer how do you feel about the pypy bit?

File: README.md (hunk 1 above)
Comment (PMSeitzer) [reply README.md:19]: 👍 lgtm

**Pull request merged**